google_reminder_api_wrapper/reminder_api.py

```diff
@@ -6,12 +6,23 @@ class ReminderApi(ReminderApiBase):
     def __init__(self):
         super().__init__()
 
-    def get(self, task_id):
+    def get(self, task_id='', clientAssignedId=''):
```

Comment:
In python I would advise to following the snake_case variable standard.

Also, would it make sense to call it `server_assigned_id` instead of `task_id` perhaps?